llpc/lower/llpcSpirvLowerTranslator.cpp

```diff
@@ -41,22 +41,32 @@
 using namespace llvm;
 using namespace Llpc;
 
-char SpirvLowerTranslator::ID = 0;
+char LegacySpirvLowerTranslator::ID = 0;
 
 // =====================================================================================================================
 // Creates the pass of translating SPIR-V to LLVM IR.
 //
 // @param stage : Shader stage
 // @param shaderInfo : Shader info for this shader
 ModulePass *Llpc::createSpirvLowerTranslator(ShaderStage stage, const PipelineShaderInfo *shaderInfo) {
-  return new SpirvLowerTranslator(stage, shaderInfo);
+  return new LegacySpirvLowerTranslator(stage, shaderInfo);
+}
+
+// =====================================================================================================================
+// Run the pass on the specified LLVM module.
+//
+// @param [in/out] M : LLVM module to be run on (empty on entry)
+// @param [in/out] AM : Analysis manager to use for this for this transformation
```

Comment:
AM -> analysisManager
The comment reads a bit weird to me (double "for this"?).